### DISCUSSION TITLE
lesson 5 and 6 mentioned a broadcast folder, but I cannot find it after deploying to anvil.

### DISCUSSION THREAD

**Discussion by oldpride** [Cyfrin/foundry-full-course-cu#260] [Q&A] [answered]
Lesson 5 and 6 mentioned a broadcast folder, but I cannot find it after deploying to anvil.

For example, in Lesson 6, I deployed contract both using command
```
forge create SimpleStorage --private-key $PRIVATE_KEY
```
and using script
```
forge script script/DeploySimpleStorage.s.sol
```
both were successful. But I don't see a broadcast folder created.

**Comment by Naman1729** [comment, marked as answer]
Hey @oldpride , You can run this :
```bash
anvil
```

then press **Ctrl + Shift + 5** to split the terminal 
Run this on new terminal :

```bash
forge script script/DeploySimpleStorage.s.sol --rpc-url $RPC_URL_ANVIL --private-key $PRIVATE_KEY_ANVIL
```
Hope this helps you !!

**Comment by oldpride** [comment]
@Naman1729 thank you for replying. I already did what you said. The extra command args in your example were default. Besides, my commands all worked for the purpose of deploying contracts.

**Reply by allwin199** [reply]
@oldpride post your code on github and send the link, it will be easy for someone to help

**Comment by Naman1729** [comment]
hey @oldpride , Have you tried to put --broadcast at the end of command . if not run this :

```bash
forge script script/DeploySimpleStorage.s.sol --rpc-url $RPC_URL_ANVIL --private-key $PRIVATE_KEY_ANVIL --broadcast
```
In case this solution does not work, kindly consider sharing your GitHub link for further discussion.

**Comment by Niferu** [comment]
Hello @oldpride 

Please remember to include below in your deploy code and also run script/deploy command with `--broadcast` flag

```solidity
        vm.startBroadcast();
        FundMe fundMe = new FundMe(priceFeed);
        vm.stopBroadcast();
```

**Reply by f3d0ss** [reply]
Actually you do not need the `--broadcast` if you just want your transaction in the `broadcast` folder without sending them on chain.
But my guess is that he forgot the `vm.startBroadcast();` as you suggested.
